packages/renderer/src/lib/preferences/PreferencesRegistriesRendering.svelte

```diff
@@ -107,6 +107,6 @@ function toggleRegistryModal(): void {
     <PreferencesRegistriesEditCreateRegistryModal
       toggleCallback="{toggleRegistryModal}"
       mode="{preferencesMode}"
-      registry="{selectedRegistry}" />
+      registry="{{source: selectedRegistry?.source, serverUrl: selectedRegistry?.serverUrl, username: selectedRegistry?.username, secret: selectedRegistry?.secret}}" />
```

Comment:
will it work to use `{...selectedRegistry}` ?

Reply:
need to check

Reply:
approving but better if we can use spread syntax as we don't need to enumerate all fields

Reply:
yep, that works, updated the PR